src/Entity/ContentType.php

```diff
@@ -1057,6 +1057,25 @@ public function getFieldType()
         return $this->fieldType;
     }
 
+    /**
+     * @return string[]
+     */
+    public function getClearOnCopyProperties(): array
+    {
+        $clearPropertyNames = [];
+
+        foreach ($this->getFieldType()->loopChildren() as $child) {
+            $extraOptions = $child->getExtraOptions();
+            $clearOnCopy = isset($extraOptions['clear_on_copy']) ? \boolval($extraOptions['clear_on_copy']) : false;
```

Comment:
Shorter solution? If $extraOptions['clear_on_copy'] is a boolean and not a string 'true'
```suggestion

            $clearOnCopy = true === $extraOptions['clear_on_copy'] ?? null;
```

Reply:
Agree